dv/uvm/core_ibex/collect_results.py

```diff
@@ -4,78 +4,217 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import argparse
+import collections
+import junit_xml
+import os.path
 import sys
-from typing import TextIO
+import yaml
+from test_run_result import TestRunResult, test_run_result_fields, \
+    check_test_run_result
```

Comment:
I think it's probably nicer to use parens than backslashes to delimit things here:
```suggestion
from test_run_result import (TestRunResult, test_run_result_fields,
                             check_test_run_result)
```